lbaas/lbaas_client.go

```diff
@@ -0,0 +1,228 @@
+package lbaas
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/go-oracle-terraform/client"
+	"github.com/hashicorp/go-oracle-terraform/opc"
+	"github.com/mitchellh/mapstructure"
+)
+
+/*
+ * LBaaSClient is the base client implementation for the Load Balancer Classic API
+ * Specialized clients are implemented for different LBaaS Service resources:
+ * - LoadBalancerClient - Load Balancers
+ * - LBaaSResourceClient - base client impl for child resources of a Load Balancer instnance:
+ *   - PolicyClient - Policies
+ *   - ListenerClient - Listeners
+ *   - OriginServerPoolClient - Origin Server Pools
+ * - SSLCertificateClient - SSL Certificates
+ */
+
+// Main Resource Content Types. Each resource has it's own ContentType
+const CONTENT_TYPE_VLBR_JSON = "application/vnd.com.oracle.oracloud.lbaas.VLBR+json"
```

Comment:
Can the casing on these be `ContentTypeVLBRJSON`, `ContentTypeListenerJSON`, etc.?

Reply:
updated const naming style to CamelCase